test/index.test.ts

```diff
@@ -302,6 +302,27 @@ test('ensure run and exec with command defined as array are working', async ():
   await compose.down({ cwd: path.join(__dirname), log: logOutput });
 });
 
+test('build accepts config as string', async (): Promise<void> => {
+  const configuration = await new Promise(function (resolve, reject) {
```

Comment:
This should allow typescript to properly type-check the `resolve` call and prevent the need for typecasting later on in this code.
```suggestion
  const configuration = await new Promise<string>(function (resolve, reject) {
```